.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Question
+about: Who, what, when, where, and how?
+
+---
+
+Hey, stop right there!
```

Comment:
Recommend making this sentence **bold** and/or leading `#` (or `##` or `###` to make font smaller) to make it a heading and highly visible.

Reply:
This should be never convert into an issue, so you only see it in the editor mode.